client/js/components/statement/publicStatementLists/DpPublicStatement.vue

```diff
@@ -133,31 +133,31 @@
    --><div class="inline-block u-2-of-3 u-1-of-1-palm">
         {{ authoredBy }}
       </div>
-    </div><!--
+      </div><!--
```

Comment:
please revert all indentation changes in this file

Reply:
If I want to revert, then I want to commit, everything again before commit, what can I do?

Reply:
It looks like You have enabled some auto-lintin on commit. Have a look on your git-commit window if you find that option. (most likely a checkbox)

Reply:
no, everytings it unchecked!

Reply:
Done in
https://github.com/demos-europe/demosplan-core/pull/2437/commits/c326ac0d93a4d61dfd5366e8f814c95f0f6cb5fe
https://github.com/demos-europe/demosplan-core/pull/2437/commits/4b57cd83a68debd4eb3996c7204afab492ae53cd